packages/cli/package.json

```diff
@@ -52,7 +52,7 @@
     "access": "public"
   },
   "dependencies": {
-    "@shiftsmartinc/feathers-generators": "^5.0.9-beta.2",
+    "@shiftsmartinc/feathers-generators": "file:packages/generators",
```

Comment:
just for testing purpose to be able to run on the local packages

Reply:
We'll circle back on whether or not we can keep it this way for our use-case.